spec/factories/sender_infos.rb

```diff
@@ -0,0 +1,23 @@
+FactoryGirl.define do
+  factory :sender_info do
```

Comment:
Albo nawet niekoniecznie enumeratorem - można to zrobić jakąś relacją polimorficzną (fajne, ale może być ryzykowne). Dlatego zostawiam Twojej jurysdykcji, takie w sumie też ujdzie.